pybess/bess.py

```diff
@@ -182,16 +183,16 @@ def _update_status(self, connectivity):
         else:
             self.status = connectivity
 
-    def connect(self, host='localhost', port=DEF_PORT):
+    def connect(self, grpc_url=DEF_GRPC_URL):
```

Comment:
Aha, this causes test failures in pybess/test_bess.py (but that seems to be the only user of the old `port=` argument so we should just fix that one up).

Reply:
Oops, thanks for figuring that out, I will fix it